lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -153,14 +153,22 @@ class TurboGraft.Remote
       initiator: @initiator
       xhr: xhr
 
-    if @refreshOnError
-      Page.refresh
-        response: xhr
-        onlyKeys: @refreshOnError
-    else if @refreshOnErrorExcept
-      Page.refresh
-        response: xhr
-        exceptKeys: @refreshOnErrorExcept
+    unless TurboGraft.hasTGAttribute(@initiator, 'tg-remote-norefresh')
```

Comment:
I'm not a big fan of using `unless` with an `else`, but that may just be personal preference.